src/components/src/ProductDisplayItem/productdisplayitem.main.less

```diff
@@ -365,47 +364,165 @@
   }
 }
 
+.product-item-container {
+  @media (min-width: @tabletWidth) {
+    display: flex;
+
+    .itemdetail-assets {
+      width: 700px;
+    }
+
+    .itemdetail-details {
+      width: auto;
+      flex-grow: 1;
+    }
+  }
+}
+
 .product-image-carousel {
+  max-width: 450px;
```

Comment:
Setting this max-width will affect the display of featured products on the category page as well. This rule needs to be more strict:

![image](https://user-images.githubusercontent.com/39800563/75158827-f4558900-56e4-11ea-834d-b820097f3e5c.png)